src/plugins/intel_gpu/src/kernel_selector/kernels/dynamic_quantize/dynamic_quantize_kernel_opt_kv_cache.cpp

```diff
@@ -118,8 +118,11 @@ JitConstants DynamicQuantizeKernelKVCache::GetJitConstants(const dynamic_quantiz
 
     const auto& input_dims = get_normalized_dims(params.inputs[0]);
     const auto total_grouped_elements = get_elements_number_per_group(params);
-    const auto per_iter_elements_number = get_per_iter_elements_number(params);
     const auto total_subgroups_number = total_grouped_elements / input_dims.back().v;
+    const auto per_iter_elements_number = get_per_iter_elements_number(params);
+    if (per_iter_elements_number == 0) {
+        throw std::runtime_error("Invalid configuration: per_iter_elements_number is zero, division by zero would occur.");
```

Comment:
Please use `OPENVINO_THROW` instead of `throw`.

Reply:
Applied it and thanks!